static/styles/zulip.scss

```diff
@@ -2656,7 +2651,7 @@ select.inline_select_topic_edit {
 
     .top-navbar-border {
         margin-left: 40px;
-        width: calc(100% - 116px);
+        width: calc(100% - 108px);
     }
     // todo: Figure out why this has to be different
     // from above at this width and resolve it
```

Comment:
@SiddharthVarshney please see this comment and adjust the width bellow as well...
(we should also delete the duplicate, commented out line here)

Reply:
Sure I will look into it